docs/examples/scf_to_cps/imputing-from-scf-to-cps.md

```diff
@@ -548,9 +548,9 @@ This transformation is particularly valuable for wealth data, where values can s
 donor_data = scf_data[predictors + imputed_variables + weights]
 receiver_data = cps_data[predictors + household_weights]
 
-donor_data, dummy_info, normalizing_params = preprocess_data(donor_data[predictors + imputed_variables], normalize=True, full_data=True)
+donor_data, normalizing_params = preprocess_data(donor_data[predictors + imputed_variables], normalize=True, full_data=True)
```

Comment:
Not introduced by this PR but lets make sure in a future one to turn off normalisation